package.json

```diff
@@ -47,7 +47,6 @@
   "devDependencies": {
     "coffeescript": "~2.4.1",
     "mocha": "~6.1.4",
-    "node-abi": "^2.19.3",
```

Comment:
@thegecko what was the purpose of that package here? I have removed this since I don't see this used anywhere.

Reply:
I added it to be able to control `node-abi` upgrades without relying on the dependency graph, happy to remove it, however.